docs/contributing.rst

```diff
@@ -187,7 +187,7 @@ Authenticators
 
 Authenticators are plugins that prove control of a domain name by solving a
 challenge provided by the ACME server. ACME currently defines three types of
-challenges: HTTP, TLS-SNI, and DNS, represented by classes in `acme.challenges`.
+challenges: HTTP, TLS-SNI (deprecated), TLS-ALPR, and DNS, represented by classes in `acme.challenges`.
```

Comment:
Four types of challenge, if we let the TLS-SNI be in the enumeration here. Or we remove it completely and forget about it? Technically this challenge is still in the spec and the servers that implement the spec, but not for long.

Reply:
Oops, thanks @adferrand!